### DISCUSSION TITLE
Emby Unknown Failure

### DISCUSSION THREAD

**Discussion by jiordanza** [tteck/Proxmox#583] [Help Needed] [answered]
I just found out this helper scripts and decided to try the emby script, but I'm getting this error. I thought it was one of my settings as I tried the advanced mode first to set my preffered config, but I also get the same error as the defaults

![chrome_5UV6YTpt75](https://user-images.githubusercontent.com/51228622/193377878-e9408b46-7856-4060-9a9f-e19b57773180.png)

**Comment by tteck** [comment, marked as answer]
For some reason they didn't produce all of the assets!
https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.7.7.0
But they did for the 4.8.0.10-beta
https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.8.0.10
So, I'll hard code the version (4.7.6.0) in for now.

**Reply by jiordanza** [reply]
Wow that was a fast reply.
I just setup my proxmox server, still learning, and I will definitely start with your scripts to play around with PVE
